### ISSUE TITLE
[Ansible] Download links for Intel Windows installers fail

### ISSUE THREAD

**Issue by EmilAlexandruStoica** [microsoft/openenclave#1617]
The Intel SGX PSW and Intel SGX DCAP Windows installers [download links](https://github.com/Microsoft/openenclave/blob/9d4addf35da3676917797b4884ca53a446cfc767/scripts/ansible/roles/windows/az-dcap-client/vars/windows.yml#L5-L6) do not work anymore (404 error is returned):

```
TASK [windows/az-dcap-client : Azure DCAP Client | Download the Intel SGX self-extracting archives and devcon.exe] 
***********************************************************************************************
failed: [astoica-win.westeurope.cloudapp.azure.com] (item={'dest': 'C:\\Windows\\Temp\\Intel_SGX_PSW.exe', 
'url': 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe'}) => 
{"changed": false, "dest": "C:\\Windows\\Temp\\Intel_SGX_PSW.exe", "item": {"dest": "C:\\Windows\\Temp\\Intel_SGX_PSW.exe", 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe"}, 
"msg": "Error downloading 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe' 
to 'C:\\Windows\\Temp\\Intel_SGX_PSW.exe': The remote server returned an error: (404) Not Found.", "status_code": 404, 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe", 
"win_get_url": {"dest": "C:\\Windows\\Temp\\Intel_SGX_PSW.exe", 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15018/Intel%20SGX%20PSW%20for%20Windows%20v2.2.100.48339.exe"}}
failed: [astoica-win.westeurope.cloudapp.azure.com] (item={'dest': 'C:\\Windows\\Temp\\Intel_SGX_DCAP.exe', 
'url': 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe'}) => 
{"changed": false, "dest": "C:\\Windows\\Temp\\Intel_SGX_DCAP.exe", "item": {"dest": "C:\\Windows\\Temp\\Intel_SGX_DCAP.exe", 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe"}, 
"msg": "Error downloading 'http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe' 
to 'C:\\Windows\\Temp\\Intel_SGX_DCAP.exe': The remote server returned an error: (404) Not Found.", "status_code": 404, 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe", 
"win_get_url": {"dest": "C:\\Windows\\Temp\\Intel_SGX_DCAP.exe", 
"url": "http://registrationcenter-download.intel.com/akdlm/irc_nas/15017/Intel%20SGX%20DCAP%20for%20Windows%20v1.0.100.48134.exe"}}
```

**Comment by CodeMonkeyLeet** [comment]
@EmilAlexandruStoica The new versions of the packages are: 

* Intel DCAP 1.1 for Windows: http://registrationcenter-download.intel.com/akdlm/irc_nas/15384/Intel%20SGX%20DCAP%20for%20Windows%20v1.1.100.49925.exe
* Intel PSW 2.3 for Windows: http://registrationcenter-download.intel.com/akdlm/irc_nas/15369/Intel%20SGX%20PSW%20for%20Windows%20v2.3.100.49777.exe

Also synced with Intel on whether the links are expected to be stable and they suggested that we cache anything we expect to download from registrationcenter, so we should probably move to that model instead of directly referencing their CDN link.

**Comment by johnkord** [comment]
This was fixed by #1718

**Comment by ionutbalutoiu** [comment]
> This was fixed by #1718

@johnkord, this issue is referring to the Windows download URLs. Your PR updated only the Linux download urls only.

> @EmilAlexandruStoica The new versions of the packages are:
> 
> * Intel DCAP 1.1 for Windows: http://registrationcenter-download.intel.com/akdlm/irc_nas/15384/Intel%20SGX%20DCAP%20for%20Windows%20v1.1.100.49925.exe
> * Intel PSW 2.3 for Windows: http://registrationcenter-download.intel.com/akdlm/irc_nas/15369/Intel%20SGX%20PSW%20for%20Windows%20v2.3.100.49777.exe
> 
> Also synced with Intel on whether the links are expected to be stable and they suggested that we cache anything we expect to download from registrationcenter, so we should probably move to that model instead of directly referencing their CDN link.

@CodeMonkeyLeet, thank-you for the new installers. I tested them and Ansible setup finishes successfully. I used our OE Jenkins storage account to cache them, and passed the storage account download URLs.

Here's [my PR](https://github.com/microsoft/openenclave/pull/1790) to have this issue solved.
